### ISSUE TITLE
[FEATURE]  information button for data displayed [3D mapping, eDNA, Seascape Genomics, etc...]

### ISSUE THREAD

**Issue by WWprado** [EPFL-ENAC/lgb-trsc#10]
A pop-up provides a brief explanation of the types of data available, with a hyperlink directing you to a dedicated page (located in the research projects in the menu bar) containing more detailed information about the specific research project.

**Comment by Guilhembp** [comment]
yes absolutely - I will write this week the text for each specific research project so that you can fill up the page and create the hyperlink

**Comment by guilbep** [comment]
@charlottegiseleweil  What do we do with this issue though ?

**Comment by charlottegiseleweil** [comment]
duplicate of #81 . closing. thanks for the flag @guilbep !
